Gems/LyShine/Assets/seedList.seed

```diff
@@ -18,11 +18,11 @@
 		</Class>
 		<Class name="SeedInfo" field="element" version="2" type="{FACC3682-2ACA-4AA4-B85A-07AD276D18A0}">
 			<Class name="AssetId" field="assetId" version="1" type="{652ED536-3402-439B-AEBE-4A5DBC554085}">
-				<Class name="AZ::Uuid" field="guid" value="{FE938F56-8238-5614-857E-25769CF225A6}" type="{E152C105-A133-4D03-BBF8-3D4B2FBA3E2A}"/>
+				<Class name="AZ::Uuid" field="guid" value="{C7BCB5E8-3E6B-5531-A3D9-CEE93E103F69}" type="{E152C105-A133-4D03-BBF8-3D4B2FBA3E2A}"/>
 				<Class name="unsigned int" field="subId" value="0" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
 			</Class>
 			<Class name="unsigned int" field="platformFlags" value="3" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
-			<Class name="AZStd::string" field="pathHint" value="shaders/lyshineui.azshader" type="{03AAAB3F-5C47-5A66-9EBC-D5FA4DB353C9}"/>
+			<Class name="AZStd::string" field="pathHint" value="lyshine/shaders/lyshineui.azshader" type="{03AAAB3F-5C47-5A66-9EBC-D5FA4DB353C9}"/>
```

Comment:
Just curious....what prompted this change? Did we recently move the shaders to a different folder? 
Also curious why the uuid for the seedinfo asset id changed....

Reply:
Yes, we moved the shader to a different folder although I don't think it was super recently.
The uuid changed because it gets constructed from file path + subId (product identifier), and the file path changed.